admin-dev/themes/new-theme/js/pages/product/components/categories/index.js

```diff
@@ -341,6 +316,22 @@ export default class CategoriesManager {
     });
   }
 
+  initAddCategoriesModal() {
+    const modalContent = $(ProductMap.categories.categoriesTemplate);
+
+    $(this.addCategoriesBtn).fancybox({
```

Comment:
I'm really not a fan of this fancybox.. It's supposed to be used for iframe and even this is not really handled well (IMHO) we have our own modal components we should use them

I made quite a refacto on them in another PR, but of course it's still pending.. (attachments one) I guess you can stick with this one until the other PR is merged

Reply:
Yes, we have 100% functional boostrap modal. Fancybox with iframe is a thing of the past...